conf/modules.yaml

```diff
@@ -1,2 +1,2 @@
 modules:
-    enabled: [eventbus, websocket, metaManager, edged, twin, dbTest]
+    enabled: [eventbus, websocket, metaManager, edged, twin, dbTest, testManager]
```

Comment:
Why enable `testManager` module by default which will increase the binary size?

Reply:
@m1093782566 test code is already imported in the binary, we are only enabling it in modules.yaml.
```
package main

import (
	.
        .
	_ "github.com/kubeedge/kubeedge/pkg/eventbus"
	_ "github.com/kubeedge/kubeedge/pkg/metamanager"
	_ "github.com/kubeedge/kubeedge/test"
	// _ "github.com/kubeedge/kubeedge/pkg/edgefunction"
)
```
But we should not enable the test module by default. 
My suggestion is that we can have a job in CI to change modules.yaml and enable test before running integration tests. @m1093782566 this solution looks good ?

Reply:
> @m1093782566 test code is already imported in the binary

Okay, I am lost :)

> But we should not enable the test module by default.
My suggestion is that we can have a job in CI to change modules.yaml and enable test before running integration tests. @m1093782566 this solution looks good ?

Sounds like a reasonable plan, can we do that? @pavan187

Reply:
@m1093782566 Okay, will change accordingly

Reply:
@m1093782566 , as discussed, integrations code is not enabling the testManager module by default. testManager module now enabled only just before running the integration suit. which is handled by the script.